README.md

```diff
@@ -1,36 +1,7 @@
-# HW03
-## Assignment
-Read in some data and graph something in R
+#Homework 3
```

Comment:
Great topic. You might want to add a space after "#" to format the title (header).